backend/src/routes/OTP/sendEmail.ts

```diff
@@ -0,0 +1,42 @@
+import Mailgun from "mailgun.js";
+import FormData from 'form-data';
+
+const mailgun = new Mailgun(FormData);
+
+export const sendEmail = async (emailAddress: string, userName: string, code: string) => {
+    if(process.env["CI"]) return;
+    const key = process.env["EMAIL_KEY"]
+    // don't send email when running tests.
+    if(key === "test") {
+        return;
+    }
+
+    const mg = mailgun.client({username: 'api', key: key || ""});
+    if(!mg){ 
+        throw new Error("invalid email key");
+    }
+
+    //console.log(`emailing ${userName} at ${emailAddress} with otp ${code}.`);
+    const senderDomain = "mg.pyrmds.app";
+
+    try {
+        const res = await mg.messages.create(`${senderDomain}`, {
+            from: `No Reply <noreply@${senderDomain}>`,
+            to: [emailAddress],
+            subject: "Pyramids Login One Time Code",
+            text: "Hi",
+            html: `
+            <div>
+                <p>Dear ${userName},</p>
+                <p>We have received a request from you to reset your password for Pyramids.</p>
+                <p>Your one time code is: <strong>${code}</strong>.</p>
+                <p>Please note that this code will expire in 60 seconds.</p>
```

Comment:
60 seconds is craaazy tho maybe 10 minutes?? 😭

Reply:
It was 5 minutes before, but I reduced it because I have testing which checks that codes are deleted after the time limit, which were delaying CI checks 😭 I can either remove the test, or we can have the CI take 10+ minutes (assuming I change to 10 mins). Which would you think would be better?

Reply:
could we have the best of both worlds using env vars? make it 5-10 mins for dev/prod, 1min for testing?

Reply:
the env var idea is nice - i implemented it so that it is 10 mins for dev/prod and 1 second for testing 🫡